README.md

```diff
@@ -7,198 +7,170 @@ A python package that creates colourblind friendly plots.
 
-   - This functions implements a method that makes histograms accesible to people with one of these three colourblind variants; deuteranopia, protanopia or tritanopia. It modifies the colour of each line, the title, legend and axes size, transparancy of the objects number of bins,  as well as the face colour.
-   
    **inputs:** y variable (a list of lists), transparency, labels of each y variable, palette colour, title, x labels, y labels and legend title.   
    **output:** a matplotlib object.
 
 
-for more details please use the the following commands:
+For more details, please use the following commands:
 
 `help(Colourblind8)` for class docstring
 
 `help(Colourblind8.modulename)` for module docstring
-  
 
-### Installation 
 
+### Installation
 
 To install `Colourblind8` via pip, please input the following into the Terminal:
 
-```pip install git+https://github.com/UBC-MDS/Colourblind8.git```
+`pip install git+https://github.com/UBC-MDS/Colourblind8.git`
 
+To upgrade to the latest version, use:
 
-### Usage 
+`pip install -- upgrade git+https://github.com/UBC-MDS/Colourblind8.git`
 
-Please note, for a more clear execution of our package, Varada Kolhatkar (Lab instructor) suggested an additional document (jupyter notebook) using our package with a test dataset, along with our tests being passed. She found this a more beneficial method of visually comparing the output with the test cases. 
-This example can be found in the `Example` folder of this repository named [`Sample_usage.ipynb`](INSERT LINK HERE). 
+### Usage
 
-To use our package it is necessary for all functions to import our package as well as matplotlib in the following manner. 
-```
-import matplotlib.pyplot as plt
-from colourblind8.colourblind8 import Colourblind8
-```
+Please note, for a more clear execution of our package, Varada Kolhatkar (UBC MDS instructor) suggested an additional document (jupyter notebook) to demonstrate how to use our package with a test dataset, along with our tests being passed. She found this a more beneficial method of visually comparing the output with the test cases.
+
+The demonstration can be found in the `Example` folder of this repository named [sample_usage.ipynb](https://github.com/UBC-MDS/Colourblind8/blob/master/example/Sample.ipynb).
 
 
-#### `plot_lines(self, x, y, alpha = 1.0, labels = None, palette = None, title = None, x_lab = None, y_lab = None, legend_title = None)` : 
+##### Step by step guide
 
-The test dataset used for this example is the following
+1. Since `Colourblind8` is built on `matplotlib`, it is necessary to import `matplotlib`, as well as `Colourblind8`. Don't forget to initialize `Colourblind8` before using.
 
 ```
-x=[1,2,3,4]
-y_list=[[1,2,3,4],[3,5,7,9],[4,7,5,9],[1,4,6,8],[4,6,7,9],[1,3,4,8],[0,4,7,2],[-1,4,7,-7],[9,7,5,3]]
+import matplotlib.pyplot as plt
+from colourblind8.colourblind8 import Colourblind8
+cb = Colourblind8() #initialize
```

Comment:
DELETE THIS LINE!! 

They can call it whatever they want.

Reply:
change made to new Readme.

Reply:
cb = Colourblind8() #initialize just this one